src/components/NewsAndArticles/index.jsx

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import './styles.css';
+import { charityAPI } from '../../clients';
+import Heading from '../Heading/index';
+
+class News extends Component {
+  state = {
+    data: {
+      heading: {},
+      description: '',
+      link: {},
+      articles: []
+    },
+    loading: false,
+    error: false,
```

Comment:
I think that th initial state of loading and errorshoud be f

Reply:
It does make sense too. We can discuss the benefit of your approach with @muubar  and @Nilegfx

Reply:
What hassan mentioned make sense in case all possible states are covered.

@elhaw what we did on your component was temporarily until we properly cover all errors and edge cases